main.config.js

```diff
@@ -201,120 +201,32 @@ export const config = {
     },
     {
       name: 'Portfolio',
+      description: '???',
+      metaTitle: 'Portfolio Keenethics',
+      metaDescription: '???',
```

Comment:
@dmytropanontko , should we have something here? Can you ask Max Savonin regarding these?

Reply:
@epaminond Yes. Max Savonin manage this thing. But for now we haven't needen text.